packages/indexer-cli/src/rules.ts

```diff
@@ -212,24 +210,23 @@ export const printIndexingRules = (
       rule => rule?.decisionBasis === IndexingDecisionBasis.OFFCHAIN,
     )
 
-    print.info(displayIndexingRules(outputFormat, onchainRules))
-    if (offchainRules) {
-      print.highlight('Offchain sync list')
-      print.info(
-        offchainRules.map(rule => {
-          return rule.identifier
-        }),
-      )
-    } else {
-      print.info(`Not syncing any subgraphs offchain`)
-    }
+    return `${displayIndexingRules(outputFormat, onchainRules)}\n${
+      offchainRules.length
+        ? [
+            chalk.bold('Offchain sync list'),
+            offchainRules.map(rule => {
+              return rule.identifier
+            }),
+          ].join('\n')
+        : chalk.dim(`Not syncing any subgraphs offchain`)
+    }`
```

Comment:
I'd suggest moving the conditional logic out of this template literal, as it is becoming a bit too convoluted.

Reply:
I moved this out to `offchainRulesDisplay` for the Display string with a comment, lmk if that is clear or should be factored out?

Reply:
I liked it! Thanks!
It's easyer to read/review as less stuff is happening in the same statement.